src/api.rs

```diff
@@ -457,6 +457,7 @@ impl Config {
     Context {
       frame_count: 0,
       limit: 0,
+      skip: 0,
```

Comment:
This feature shouldn't live in the encoder library but only in the utility binary.

Reply:
Pushed the new changes